_dashboards/search-telemetry.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Search telemetry
+nav_order: 30
+---
+
+
+# About search telemetry
+
+You can use search telemetry to analyze performance for success or failed search requests in OpenSearch Dashboards. OpenSearch stores telemetry data in the `.kibana_1` index.
```

Comment:
"to analyze performance for success or failed search requests" reads a bit awkwardly. Would it make sense to say "to identify successful or failed search requests"?